scripts/pull_github_pr.sh

```diff
@@ -17,7 +17,16 @@ if [[ ( -z "$GITHUB_LOGIN" || -z "$GITHUB_TOKEN" ) && -f "$gh_hosts" ]]; then
 	GITHUB_TOKEN=$(awk '/oauth_token:/ { print $2 }' "$gh_hosts")
 fi
 
-if [[ $# != 1 ]]; then
+if [[ -z "$JENKINS_USERNAME" || -z "$JENKINS_API_TOKEN" ]]; then
+  echo """
```

Comment:
nitpick, this is a shell, not Python, so triple-quote is meaningless - actually the first two quotes are an empty string and then you have one more real quote. So the triple quotes is just the same as one quotes.

Note that using quotes the way you did will print out potentially-unwanted indentation as part of the message. The traditional shell trick to print a long message without indentation is a so-called "**here document**":

```bash
cat <<-END
<tab>hello
<tab>hi
END
```
but sadly, this was invented in the days that only tabs were used for indentation so it doesn't work with spaces so it's worthless for us.